File: homework_3/task3.js
/* Given an array of strings and numbers. 
Print the number of integers and the number 
of strings in the array */

"use strict";

function countNum(array) {
    let n = 0, s = 0, txt = "";
    for (let i = 0; i < array.length; i++) {
        if (typeof (array[i]) === "number") {
            n++;
        } else {
            s++;
        }
    }
    return `Numbers: ${n}, Strings: ${s}`;
}
//test
let a1 = "[1, \"10\", \"hi\", 2, 3] => " + countNum([1, "10", "hi", 2, 3])
let a2 = "[1, 4, \"i am a string\", \"456\"] => " + countNum([1, 4, "i am a string", "456"])
console.log(a1);
console.log(a2);

File: Homeworks/homework_03/archive/task3.py
def getNumbers(li):
    num, st= 0, 0
    for x in li:
        if isinstance(x, str): st += 1
        else: num += 1
    return f"Numbers: {num}, Strings: {st}"
print(getNumbers([1, "10", "hi", 2,3 ]))
print(getNumbers([1, 4, "I am a string", "456"]))